resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClusterManager.scala

```diff
@@ -53,4 +54,6 @@ private[spark] class YarnClusterManager extends ExternalClusterManager {
   override def initialize(scheduler: TaskScheduler, backend: SchedulerBackend): Unit = {
     scheduler.asInstanceOf[TaskSchedulerImpl].initialize(backend)
   }
+  override def createShuffleServiceAddressProvider(): ShuffleServiceAddressProvider =
```

Comment:
FYI neither this nor the Mesos version of this method compile.

Reply:
I was building distribution without the -Pyarn and -Pmesos commands :) I'll fix that